framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2665,12 +2666,13 @@ public Void visitCatch(CatchTree tree, Void p) {
 
     /**
      * Checks the type of a thrown exception. Subclasses should override
-     * checkThrownExpression(ThrowTree tree) rather than this method to change the behavior of this
-     * check.
+     * checkThrownExpression(ThrowTree tree, MethodTree mtree) rather than this method to change the
+     * behavior of this check.
      */
     @Override
     public Void visitThrow(ThrowTree tree, Void p) {
-        checkThrownExpression(tree);
+        MethodTree mtree = this.methodTree;
+        checkThrownExpression(tree, mtree);
```

Comment:
Why pass the field value as a parameter? `checkThrownExpression` is an instance method, so it can access the field if it so desires.

Reply:
Already undo the changes.